docs/contracts/v4/concepts/01-v4-vs-v3.mdx

```diff
@@ -1,30 +1,68 @@
 ---
-title: v4-versus-v3
+title: V4 vs V3
 ---
 
-While Uniswap v4's underlying concetrated liquidity is the same as Uniswap v3, there are some key differences in the architecture
-and accounting.
+While Uniswap v4's underlying concentrated liquidity is the same as Uniswap v3,
+there are some key differences in the architecture and accounting.
 
 # Singleton Design
 
 ### Pool Creation
 
-(TODO: talk about singleton vs factory, state vs contract/V3Pool)
+V4: The singleton contract facilitates the creation of a pool and 
+also stores its state. This pattern reduces costs when creating a pool
+and doing multi-hop swaps.
```

Comment:
```suggestion
and doing multi-hop swaps. Because pools are _contract state_ and not entirely new _contracts_ themselves, pool creation is significantly cheaper.
```